src/managedwriter/stream_connection.ts

```diff
@@ -245,6 +253,27 @@ export class StreamConnection extends EventEmitter {
     return null;
   }
 
+  private resendAllPendingWrites() {
+    let pw = this._pendingWrites.pop();
+    while (pw) {
+      this.send(pw);
```

Comment:
what's the behavior if send fails?  do you need to mark the write bad?

Reply:
after a number of attempts it is marked as failed here:
* https://github.com/googleapis/nodejs-bigquery-storage/blob/474e2377b22a2da33167097a6e1f6409ec081774/src/managedwriter/stream_connection.ts#L319

and if write retries are not enabled, it is nack here:
* https://github.com/googleapis/nodejs-bigquery-storage/blob/474e2377b22a2da33167097a6e1f6409ec081774/src/managedwriter/stream_connection.ts#L117